sdk/src/logs/logger_provider.cc

```diff
@@ -88,9 +88,20 @@ nostd::shared_ptr<opentelemetry::logs::Logger> LoggerProvider::GetLogger(
   }
   */
 
-  // If no logger with that name exists yet, create it and add it to the map of loggers
-  auto lib = instrumentationlibrary::InstrumentationLibrary::Create(library_name, library_version,
-                                                                    schema_url);
+  // For log sources which define a logger name (e.g. Java Logger Name) the Logger Name SHOULD be
```

Comment:
nit: put a link to the spec here instead of copy the text?

Reply:
Done